### PR TITLE
Logs minor and major crimes

### PR DESCRIPTION
**Pull request by nfreader**
:cl: atlanta_ned
:add: Simply adds minor and major crimes to the records.html investigate log
/:cl:

[why]: More logs for the log gods and I'm curious to see what people are being arrested for.

### CODE REVIEW

File: code/game/machinery/computer/security.dm [hunk 1]
```diff
@@ -667,6 +668,7 @@ What a mess.*/
 								return
 							var/crime = GLOB.data_core.createCrimeEntry(t1, t2, authenticated, station_time_timestamp())
 							GLOB.data_core.addMajorCrime(active1.fields["id"], crime)
+							investigate_log("New Major Crime: <strong>[t1]</strong>: [t2] | Added to [active1.fields["name"]] by [usr.name] ([usr.key])", INVESTIGATE_RECORDS)
```
Comment (ShizCalev) [inline code/game/machinery/computer/security.dm]: [key_name(usr)]

File: code/game/machinery/computer/security.dm [hunk 2]
```diff
@@ -653,6 +653,7 @@ What a mess.*/
 								return
 							var/crime = GLOB.data_core.createCrimeEntry(t1, t2, authenticated, station_time_timestamp())
 							GLOB.data_core.addMinorCrime(active1.fields["id"], crime)
+							investigate_log("New Minor Crime: <strong>[t1]</strong>: [t2] | Added to [active1.fields["name"]] by [usr.name] ([usr.key])", INVESTIGATE_RECORDS)
```
Comment (ShizCalev) [inline code/game/machinery/computer/security.dm]: [key_name(usr)]

File: code/modules/mob/living/carbon/human/human.dm [hunk 3]
```diff
@@ -433,6 +433,7 @@
 														return
 													var/crime = GLOB.data_core.createCrimeEntry(t1, t2, allowed_access, station_time_timestamp())
 													GLOB.data_core.addMinorCrime(R.fields["id"], crime)
+													investigate_log("New Minor Crime: <strong>[t1]</strong>: [t2] | Added to [R.fields["name"]] by [usr.name] ([usr.key])", INVESTIGATE_RECORDS)
```
Comment (ShizCalev) [inline code/modules/mob/living/carbon/human/human.dm]: [key_name(usr)]

File: code/modules/mob/living/carbon/human/human.dm [hunk 4]
```diff
@@ -448,6 +449,7 @@
 														return
 													var/crime = GLOB.data_core.createCrimeEntry(t1, t2, allowed_access, station_time_timestamp())
 													GLOB.data_core.addMajorCrime(R.fields["id"], crime)
+													investigate_log("New Major Crime: <strong>[t1]</strong>: [t2] | Added to [R.fields["name"]] by [usr.name] ([usr.key])", INVESTIGATE_RECORDS)
```
Comment (ShizCalev) [inline code/modules/mob/living/carbon/human/human.dm]: [key_name(usr)]

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Review by ShizCalev — DISMISSED** [review]

File: code/modules/mob/living/carbon/human/human.dm [hunk 5]
```diff
@@ -391,7 +391,7 @@
 										if(R)
 											if(H.canUseHUD())
 												if(istype(H.glasses, /obj/item/clothing/glasses/hud/security) || istype(H.getorganslot(ORGAN_SLOT_HUD), /obj/item/organ/cyberimp/eyes/hud/security))
-													investigate_log("[src.key] has been set from [R.fields["criminal"]] to [setcriminal] by [usr.name] ([usr.key]).", INVESTIGATE_RECORDS)
+													investigate_log("[src.key] has been set from [R.fields["criminal"]] to [setcriminal] by [key_name(usr)].", INVESTIGATE_RECORDS)
```
Comment (ShizCalev) [inline code/modules/mob/living/carbon/human/human.dm]: [key_name(src)] at the start too, please.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Comment by nfreader** [comment]
Also cleaned up a few other places where `key_name` should be used ^_^ I got that consistency HYPE 🎉

**Comment by ExcessiveUseOfCobblestone** [comment]
hey cutie ;)

**Review by ShizCalev — APPROVED** [review]

**Comment by Iamgoofball** [comment]
Did you fix the thing where the stats site would be missing stats on some stuff that we found when going over syndie items? You said it was on your end and I made sure the code wasn't dropping stats for any reason, we removed the stupid black box destruction deleting stats ages ago.

Because I've been asking for months if you fixed that, and stat gathering is useless if the stats can get shot down in any balance debate with "well it missed a ton of rounds so its worthless data"

**Comment by ShizCalev** [comment]
@cacogen why downvote?

**Pull request merged**